modules/configuring-the-model-registry-component.adoc

```diff
@@ -0,0 +1,124 @@
+:_module-type: PROCEDURE
+
+[id='configuring-the-model-registry-component_{context}']
+= Configuring the model registry component
+
+[role='_abstract']
+To configure the model registry feature for your data scientists to use in {productname-short}, you must install the following Operators in OpenShift, and then enable the `modelregistry` component in {productname-short}.
+
+* Red Hat Authorino Operator (Technical Preview)
+* Red Hat OpenShift Serverless Operator
+* Red Hat OpenShift Service Mesh Operator
+ifdef::upstream[]
+* Open Data Hub Operator version 2
+endif::[]
+ifndef::upstream[]
+* {productname-long} Operator version 2.14+
+endif::[]
+
+ifdef::upstream[]
+[IMPORTANT]
+====
+You must install the required Operators in the listed order. You should install the Open Data Hub Operator only after the other listed Operators are installed and in a running state.
+
+For more information about installing Operators in OpenShift, see link:https://docs.redhat.com/en/documentation/openshift_container_platform/{ocp-latest-version}/html/operators/administrator-tasks#olm-adding-operators-to-a-cluster[Adding Operators to a cluster].
+
+For information about installing the Open Hub Operator, see link:{odhdocshome}/installing-open-data-hub/#installing-the-odh-operator-v2_installv2[Installing Open Data Hub Operator version 2].
```

Comment:
For more information about installing the Open **Data** Hub Operator..

Reply:
Thank you! Updated.